deployment/deployment_utils.sh

```diff
@@ -38,6 +38,7 @@ build_directories() {
   echo "Copying general files."
   cp -r ./dataland-frontend/dist ./docker-compose.yml ./dataland-inbound-proxy/ ./dataland-inbound-admin-proxy/ ./dataland-frontend/default.conf "$target_dir"
   cp -r ./dataland-keycloak/dataland_theme/login/dist "$target_dir"/dataland-keycloak/dataland_theme/login
```

Comment:
btw I noticed some duplicate copy statements here

and furthermore, the logical grouping of the copy tasks is not valid anymore

and why do we need the backend jar, if we don't even use it on the server?